README.md

```diff
@@ -54,3 +54,12 @@ This imports the USWDS, pulls in the NJ-specific components and styles, and save
 This builds USWDS styles, builds the Fractal docs, and then deploys them to the gh-pages branch.
 
 Note: do not push directly to the gh-pages branch.
+
+## Releasing a new version to NPM
+
+1. Go to the Draft Release Github Action https://github.com/newjersey/njwds/actions/workflows/draft-release.yml
+2. Click Run workflow dropdown, keep the branch set to `main`, and update the Semver Level based on what has changed since the previous release (see here for details: https://semver.org/). Click green "Run workflow" button.
```

Comment:
I think maybe this should be `Click the "Run workflow" dropdown` and `Click the green...`